src/components/letter/LetterDetailSection.tsx

```
@@ -136,54 +134,49 @@ export default function LetterDetailSection({
-        const clonedEl = doc.querySelector(
-          "[data-letter-card]"
-        ) as HTMLElement | null;
+        const clonedEl = doc.querySelector('[data-letter-card]') as HTMLElement | null;
 
         if (!clonedEl) return;
 
         // 기존 shadow 제거해야지 안에 회색 비침 현상 방지
-        clonedEl.style.boxShadow = "none";
-        (clonedEl.style as any).webkitBoxShadow = "none";
+        clonedEl.style.boxShadow = 'none';
+        (clonedEl.style as any).webkitBoxShadow = 'none';
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] 'e' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Empty block statement.

(no-empty)